### ISSUE TITLE
CentrifugeConnectEvent has not data property

### ISSUE THREAD

**Issue by shalom-aviv** [centrifugal/centrifuge-swift#66]
[Delegate.swift](https://github.com/centrifugal/centrifuge-swift/blob/master/Sources/SwiftCentrifuge/Delegate.swift):
```swift
//...
public struct CentrifugeConnectEvent {
    public var client: String
}
//...
```

Specification [Connect result fields](https://centrifugal.dev/docs/server/proxy#connect-result-fields):

Field | Type | Optional | Description
-- | -- | -- | --
data | JSON | yes | a custom data to send to the client in connect command response.

In Java all is OK ;) 

[ConnectEvent.java](https://github.com/centrifugal/centrifuge-java/blob/master/centrifuge/src/main/java/io/github/centrifugal/centrifuge/ConnectEvent.java)
```java
package io.github.centrifugal.centrifuge;

public class ConnectEvent {
    public String getClient() {
        return client;
    }

    void setClient(String client) {
        this.client = client;
    }

    public byte[] getData() {
        return data;
    }

    void setData(byte[] data) {
        this.data = data;
    }

    private String client;
    private byte[] data;
}
```

The same situation in dart library, but not in swift

**Comment by FZambia** [comment]
Hello @shalom-aviv  

Thanks, yeah, I noticed this and fixed while working on #58  ([code](https://github.com/centrifugal/centrifuge-swift/blob/new_protocol/Sources/SwiftCentrifuge/Delegate.swift#L13)). But that PR changes lots of things and will land a bit later (part of https://github.com/centrifugal/centrifugo/issues/500) – so for now we still need to support this field separately in current master.

Do you want to send PR supporting this? If not - I'll be able to add the missing field at some point (hopefully this week).

**Comment by shalom-aviv** [comment]
Please review my changes in pull request:
https://github.com/centrifugal/centrifuge-swift/pull/67

Thank you

**Comment by FZambia** [comment]
Thanks! Just released [0.4.6](https://github.com/centrifugal/centrifuge-swift/releases/tag/0.4.6)

**Comment by shalom-aviv** [comment]
Awesome! 
You are very speedy!
Thanks
